packages/browserify/README.md

```diff
@@ -9,26 +9,25 @@ are defined in [SES][SesGithub] containers. It aims to reduce
 the risk of malicious code in the app dependency graph, known
 as "software supplychain attacks".
 
+> For an overview of LavaMoat tools see [the main README](https://github.com/LavaMoat/LavaMoat/tree/main/README.md)
+
+
 [BrowserifyGithub]: https://github.com/browserify/browserify
 [SesGithub]: https://github.com/agoric/SES
 
 ## Anatomy of a LavaMoat bundle
 
-The `lavamoat-browserify` plugin replaces the last internal build step of the browserify [compiler pipeline](https://github.com/browserify/browserify-handbook#compiler-pipeline). This step takes all the modules and their metadata and outputs the final bundle content, including the kernel and LavaMoat policy file.
+The `lavamoat-browserify` plugin replaces the last internal build step of the browserify [compiler pipeline](https://github.com/browserify/browserify-handbook#compiler-pipeline). This step takes all the modules and their metadata and outputs the final bundle content, including the LavaMoat kernel. It also generates the LavaMoat policy file.
 
 LavaMoat builds differ from standard browserify builds in that they:
 
-1. Include the app-specified LavaMoat policy
-
-This tells the kernel what execution environment each module should be instantiated with, and what other modules may be brought in as dependencies.
-
-2. Use a custom LavaMoat kernel
-
-This kernel enforces the LavaMoat policy. When requested, a module is initialized, usually by evaluation inside a SES container.
+1. Uses `lockdown()` from [SES][SesGithub] to prevent tampering with the execution environment.
+   Thanks to lockdown, prototype-pollution attacks are neutralized. It's also a prerequisite to code isolation. 
+2. Uses SES Compartments to isolate each package's execution.
+   Packages don't share references to anything unless explicitly passed in or allowed by policy. Custom LavaMoat kernel handles the `require()` calls in the resulting bundle. When required, a module is initialized, usually by evaluation inside a SES container.
+3. Enforces the app-specified LavaMoat policy.
```

Comment:
i would rephrase:

```
Enforces the app-specified LavaMoat policy. The policy specifies what execution environment each package should load with which means: what builtin APIs should it be exposed to, and what other packages can it require/import.
```

Reply:
anything you accept in this comment should be changed in the other place this sentence shows at as well

Reply:
Took it, but kept "runs" instead of "loads" as it's more true.